lambda-src/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "slacknotification",
+  "version": "1.0.0",
+  "description": "Sends a message to a Slack channel",
+  "main": "index.js",
+  "scripts": {
+    "test": "_mocha -u bdd --timeout 999999 --colors ../test"
+  },
+  "author": "Home Office CDP team",
+  "license": "MIT",
+  "dependencies": {
+    "request": "^2.88.0",
+    "request-promise-native": "^1.0.7"
+  },
+  "devDependencies": {
+    "bluebird": "^3.5.4",
+    "chai": "^4.2.0",
+    "mocha": "^6.1.4",
+    "request-promise-native": "^1.0.7",
```

Comment:
is it a dev or prod dependency? (appears in both, which may have odd side effects)

Reply:
Good spot. It's a prod dependency so I'll remove it from the dev one

Reply:
Regarding linting, I'll add this as a requirement to CDMP-2337 (pipeline for deploying the budget alerting changes) if that's ok for now?